github4s/jvm/src/test/scala/github4s/utils/MockGithubApiServer.scala

```diff
@@ -598,29 +598,65 @@ trait MockGithubApiServer extends MockServerService with FakeResponses with Test
          """.stripMargin)))
     .respond(response.withStatusCode(createdStatusCode).withBody(validCreatePullRequest))
 
-  //Issues >> list
+  //PullRequests >> listReviews
   mockServer
     .when(
       request
         .withMethod("GET")
-        .withPath(s"/repos/$validRepoOwner/$validRepoName/issues")
+        .withPath(s"/repos/$validRepoOwner/$validRepoName/pulls/$validPullRequestNumber/reviews")
         .withHeader("Authorization", tokenHeader))
-    .respond(response.withStatusCode(okStatusCode).withBody(listIssuesValidResponse))
+    .respond(response.withStatusCode(okStatusCode).withBody(listReviewsValidResponse))
 
   mockServer
     .when(
       request
         .withMethod("GET")
-        .withPath(s"/repos/$validRepoOwner/$validRepoName/issues")
+        .withPath(s"/repos/$validRepoOwner/$validRepoName/pulls/$validPullRequestNumber/reviews")
         .withHeader(not("Authorization")))
     .respond(response.withStatusCode(unauthorizedStatusCode).withBody(unauthorizedResponse))
 
   mockServer
     .when(
       request
         .withMethod("GET")
+        .withPath(s"/repos/$validRepoOwner/$invalidRepoName/pulls/$validPullRequestNumber/reviews")
+        .withHeader("Authorization", tokenHeader))
+    .respond(response.withStatusCode(notFoundStatusCode).withBody(notFoundResponse))
+
+  //PullRequests >> getReview
+  mockServer
+    .when(
+      request
+        .withMethod("GET")
+        .withPath(
+          s"/repos/$validRepoOwner/$validRepoName/pulls/$validPullRequestNumber/reviews/$validPullRequestReviewNumber")
+        .withHeader("Authorization", tokenHeader))
+    .respond(response.withStatusCode(okStatusCode).withBody(getReviewValidResponse))
+
+  mockServer
+    .when(
+      request
+        .withMethod("GET")
+        .withPath(
+          s"/repos/$validRepoOwner/$invalidRepoName/pulls/$validPullRequestNumber/reviews/$validPullRequestReviewNumber")
+        .withHeader("Authorization", tokenHeader))
+    .respond(response.withStatusCode(notFoundStatusCode).withBody(notFoundResponse))
+
+  //Issues >> list
+  mockServer
+    .when(
+      request
+        .withMethod("GET")
+        .withPath(s"/repos/$validRepoOwner/$validRepoName/issues")
+        .withHeader("Authorization", tokenHeader))
+    .respond(response.withStatusCode(okStatusCode).withBody(listIssuesValidResponse))
+
+  mockServer
+    .when(
+      request
+        .withMethod("GET")
         .withPath(s"/repos/$validRepoOwner/$invalidRepoName/issues")
-        .withHeader(not("Authorization")))
+        .withHeader("Authorization", tokenHeader))
     .respond(response.withStatusCode(notFoundStatusCode).withBody(notFoundResponse))
```

Comment:
Hmmm I think things got mixed up because Issues >> List is testing for request without auth token: https://github.com/47deg/github4s/blob/master/github4s/jvm/src/test/scala/github4s/unit/ApiSpec.scala#L778-L782